src/applications/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -27,6 +27,18 @@ export const APPEAL_TYPES = {
 
 export const appealTypes = Object.values(APPEAL_TYPES);
 
+export const programAreaMap = {
+  compensation: 'disability compensation',
+  pension: 'pension',
+  insurance: 'insurance',
+  loan_guaranty: 'loan guaranty', // eslint-disable-line camelcase
+  education: 'education',
+  vre: 'vocational rehabilitation and employment',
```

Comment:
I think VRE stands for "Veteran Readiness and Employment" now

Reply:
Have you been able to verify that? I can make that change here if you'd like. Or you can update it when you are sure.

Reply:
Yeah, you can leave this unchanged. I contacted the content team, and we can let them figure it out - ref: https://dsva.slack.com/archives/C01K37HRUAH/p1615478799034000